src/test/java/com/iota/iri/profiling/RocksDbSizeProfiling.java

```diff
@@ -0,0 +1,141 @@
+package com.iota.iri.profiling;
+
+import com.iota.iri.conf.BaseIotaConfig;
+import com.iota.iri.controllers.*;
+import com.iota.iri.model.Hash;
+import com.iota.iri.model.persistables.Transaction;
+import com.iota.iri.storage.Tangle;
+import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RocksDbSizeProfiling {
+
+    private static final Logger log = LoggerFactory.getLogger(RocksDbSizeProfiling.class);
+
+    @Test
+    public void profileDBSectionsSize() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'profileDBSectionsSize()' has a cyclomatic complexity of 15.](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4804250)